domain/access/bootstrap/bootstrap.go

```diff
@@ -33,6 +33,8 @@ func AddUser(name string, access permission.AccessSpec) (user.UUID, internaldata
 		return "", bootstrapErr(errors.Annotatef(usererrors.UserNameNotValid, "%q", name))
 	}
 
+	external := !names.NewUserTag(name).IsLocal()
```

Comment:
`AddUser` is not called anywhere. Let's delete it along with the tests.

Reply:
Removed.